### ISSUE TITLE
[REC] Evoker/Devastation - Hekili occasionally offers Disintegrate reccomendations below 3 essence.

### ISSUE THREAD

**Issue by Fyurie** [Hekili/hekili#3451]
### Before You Begin

- [X] I confirm that I have downloaded the latest version of the addon.
- [X] I am not playing on a private server.
- [X] I checked for an [existing, open ticket](https://github.com/Hekili/hekili/labels/recommendation) for this issue and was not able to find one.
- [X] I edited the title of this issue (above) so that it describes the issue I am reporting.
- [X] I am reporting an issue with the default priority included with the specialization (imported or edited priorities are not supported).

### Describe the Issue

It's a little hard to explain well, I guess.

Occasionally during (typically) single-target gameplay, Hekili will attempt to recommend a Disintegrate as your next action despite not having 3 essence or any Essence Burst procs. This results in you effectively standing dead still and taking no actions until you either cast an LF/AS to try to fish for a proc, or regenerate to 3 essence.

I haven't noticed any particular set of conditions that makes it happen, since it will sometimes still behave as normal without a proc/3 essence and use LF to fish for it. Hopefully the snapshot can shed some light.

### How to Reproduce

1. Log in to a Devastation Evoker.
2. Pick your favorite recommended talent build from Wowhead/Archon/etc.
3. Beat on a dummy or otherwise bulky target.
4. Occasionally, you will be offered the aforementioned recommendation.

### Snapshot (Link)

https://pastebin.com/SyFLRjA1

### Raidbots Sim Report (Link)

https://www.raidbots.com/simbot/report/jPXqrv2t8Y35SATT897xYF

**Comment by Hekili** [comment]
Looks like there's a missing Azure Strike at the end of the `st` list in your priority, likely due to an extra space in the priority.  There's *possibly* an additional complication that you were estimated to be 2.95 essence in your snapshot, which *probably* shouldn't have resulted in waiting 3 seconds to cast Disintegrate.

The sim priority wouldn't have used LF to fish, because you're in Dragonrage at the time and Fire Breath's going to be on CD for more than 5 seconds.

`# Cast LF outside of DR, In DR only cast with Iridescence.`
` - this entry's criteria FAILS: ! buff.dragonrage.up[true] | ( buff.iridescence_red.remains[0.00] > execute_time[1.78] | ! talent.engulfing_blaze.enabled[true] | buff.iridescence_blue.up[false] | buff.burnout.up[false] | buff.leaping_flames.up[true] & cooldown.fire_breath.remains[12.04] <= 5 ) & active_enemies[1.00] = 1`

tl;dr -- I'll fix the missing Azure Strike entry in the next release.
